keyboards/cannonkeys/satisfaction75/satisfaction75.c

```diff
@@ -101,13 +102,65 @@ void raw_hid_receive( uint8_t *data, uint8_t length )
 				command_data[3] = (value >> 8 ) & 0xFF;
 				command_data[4] = value & 0xFF;
 			}
+      else if ( command_data[0] == id_oled_default_mode )
+			{
+        uint8_t default_oled = eeprom_read_byte((uint8_t*)DYNAMIC_KEYMAP_DEFAULT_OLED);
+        command_data[1] = default_oled;
+      }
+      else if ( command_data[0] == id_oled_mode )
+			{
+        command_data[1] = oled_mode;
+      }
+
+      else if ( command_data[0] == id_encoder_modes )
+			{
+        command_data[1] = enabled_encoder_modes;
+      }
+      else if ( command_data[0] == id_encoder_custom )
```

Comment:
Did you mean to comment these out?

Reply:
Yeah, the code on VIA side isn't quite done yet, but I want this comment here for a reminder of how i will need to implement things.